src/layouts/Folders/components/MenuDropdownButton.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  ButtonGroup,
+  IconButton,
+  Icon,
+  Menu as ChakraMenu,
+  Text,
+} from "@chakra-ui/react"
+import { Button, Menu } from "@opengovsg/design-system-react"
+import { ContextMenu } from "components/ContextMenu"
+import { useState } from "react"
+import {
+  BiChevronDown,
+  BiChevronUp,
+  BiFileBlank,
+  BiFolder,
+} from "react-icons/bi"
+import { useRouteMatch, Link as RouterLink } from "react-router-dom"
+
+export const MenuDropdownButton = (): JSX.Element => {
+  const { url } = useRouteMatch()
+  const [isCreatePage, setIsCreatePage] = useState(false)
+
+  return (
```

Comment:
Is this toggle behaviour intended? I was under the impression that clicking on one of the dropdown options would immediately bring up the appropriate modal, but users need to select an option from the dropdown then click the button - just checking whether this is intended

Reply:
hmm this is a good point - i assumed the current impl as it felt the most natural (otherwise it'd be 2 separate buttons) but will clarify with @NatMaeTan